### ISSUE TITLE
Report invalid mappings

### ISSUE THREAD

**Issue by alessiostalla** [Strumenta/at-strumenta-ast-typescript#20]
```typescript
class SomeNode extends Node {
    @Mapped("doesntExist")
    someProperty: Node;
}
```

The above doesn't report any error or warning.
